Preparation/MyStack/MyStackInter.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections;
+
+namespace MyStack
+{
+    public class MyStack
+    {
+        public static readonly int maxsize = 10;
+        public int topstack;
+        public int[] arrayint = new int[maxsize];
+
+        public MyStack()
+        {
+            topstack = 0;
+        }
+
+        public bool Add(int add)
+        {
+            if (topstack >= maxsize)
+            {
+                Console.WriteLine("Stack Overfllow");
+                return false;
+            }
+            else
+            {
+                arrayint[++topstack] = add;
+                return true;
+            }
+        }
+
+        public void SeeLastInter()
+        {
+            if (topstack < 0)
+            {
+                Console.WriteLine("Stack is null");
+            }
+            Console.WriteLine($"Top element is  {topstack}");
+        }
+
+        public int Delete()
```

Comment:
Pop. Please stick to the conventional naming. For stack - push, pop, peek. For queue - enqueue, dequeue, and so on.
General advice: get to know the area you are working with and use the appropriate language.